DC-rook

```diff
@@ -5,7 +5,7 @@
 ##
 ## Basics
 MAIN="MAIN.susestorage.xml"
-ROOTID=book-storage-rook
+ROOTID=rook-guide
```

Comment:
This is the only one I consider odd. But I think it's for the best. It's not going to last long anyway 👀